requirements.txt

```diff
@@ -1,3 +1,3 @@
-torch>=2.1.0
-lightning @ git+https://github.com/Lightning-AI/lightning@2b6b594dabe668752183f5ab8a1d93d0904c4d32
+torch==2.2.0.dev20231026
+lightning @ git+https://github.com/Lightning-AI/lightning@c524c0b01bba46b7ffb6b0668242d92084cdd44b
```

Comment:
This doesn't include lightning.data no? It would need to be added as an extra requirement.